### ISSUE TITLE
Notification sound option picker should select a sane default

### ISSUE THREAD

**Issue by gergles** [avaidyam/Parrot#102]
I imagine most people will want to use one of the system notification sounds for their chat notifications, so perhaps the file picker for those should start in /System/Library/Sounds instead of ~/Documents.

Also, adding a preference to have a default notification sound for any chats that don't have one specified might be nice.

**Comment by avaidyam** [comment]
Good point - I'll design a quick control to mimic how Messages does it. `/System/Library/PrivateFrameworks/ToneLibrary.framework/Resources` also has all of the "modern" contact sounds heard on iOS and in Messages on macOS. I've thought about what to do with that; should we copy them over at runtime or copy them over at build time (which increases the binary size by a fair amount - 10MB), or just ignore them?
